### PR TITLE
Fix: Rosetta code day of the week: Incorrect function name in text strings

### LINKED ISSUE #39379: rosetta code day of the week: Incorrect function name in text strings
The last three tests are using `findChristmasSunday` as the function name for the text output (not the actual tests). The function name should be `findXmasSunday`. This is confusing and should be fixed.

This is a simple PR so I will label it first time only.

Challenge:
https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/day-of-the-week

Challenge file:
https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md

### PR DESCRIPTION
**Pull request by amygurski**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39379

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md:28]: Just noticed, this text was missing a closing parentheses for the function call.
```suggestion
  - text: <code>findXmasSunday(2008, 2121)</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@amygurski Since we are making a change to the text, let's fix the other problem I pointed out.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md:30]:
```suggestion
  - text: <code>findXmasSunday(2008, 2121)</code> should return [2011, 2016, 2022, 2033, 2039, 2044, 2050, 2061, 2067, 2072, 2078, 2089, 2095, 2101, 2107, 2112, 2118]
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Let us not forget this one.

Also, @RandellDawson , should the expected return not also be within `<code>` tags?

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md
Comment (RandellDawson) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md:30]: @Sky020 I did not catch that.  Thanks.

>Also, @RandellDawson , should the expected return not also be within <code> tags?

Yes, I did not notice those either.

Comment (amygurski) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md]: I added the <code> tags to the two returns and squashed the commits for a cleaner commit history. 
let me know if you see/want anything else!

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
LGTM

**Review by naomi-lgbt — APPROVED** [review]
LGTM!

**Comment by RandellDawson** [comment]
@Sky020 Just need your final approval before merging.

@nhcarrigan Since your #39384 makes similar changes and this PR was first, you can close yours.

**Comment by nhcarrigan** [comment]
@RandellDawson ah sorry - #39384 is not my PR 😅

**Comment by RandellDawson** [comment]
@nhcarrigan That is what I get for glancing at the username.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 .  Thanks, _amygurski for the PR.

**Comment by RandellDawson** [comment]
Congrats @amygurski on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**
